src/scss/components/paragraphs/_chart.scss

```diff
@@ -0,0 +1,25 @@
+$_chart-aspect-ratio: 1.78; // 16/9
```

Comment:
You could use `$_chart-aspect-ratio: 16/9;` sass math here to get more decimals and accuracy.